ios_ui/MovieRec/MovieImages.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MovieImages.swift
+//  MovieRec
+//
+//  Created by Yujin Cho on 8/20/17.
+//  Copyright © 2017 Yujin Cho. All rights reserved.
+//
+
+import UIKit
+import Kingfisher
+
```

Comment:
For general purposes try to follow examples of MVVM, or even VIPER if you can. Generally stuff like parsing can be placed in some Service object, and fetching and retrieving objects should be placed in some object thats called a ViewModel or Interactor.